system/shterminal.py

```diff
@@ -12,7 +12,13 @@
     import dummyui as ui
     from .dummyobjc_util import *
 
-from .shcommon import CTRL_KEY_FLAG
+from .shcommon import CTRL_KEY_FLAG, PY3
+
+
+if PY3:
+	# rename string types
+	unicode = str
+	str = bytes
```

Comment:
```python
try:
    unicode        # Python 2
except NameError:
    unicode = str  # Python 3
```

Reply:
Ok, i get your point about `try: name except NameError: name = ...`.
I just dislike using `try: ... Except` if i already know that i have to use python3 ;)
Though i probably should do these changes since you likely have way more experience with python3 than me.

Reply:
Python porting best practice [use feature detection instead of version detection](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)

Reply:
@cclauss thanks for the link. This is my first time porting something to python3 so i am a bit inexperienced ;)